sycl/doc/GlobalObjectsInRuntime.md

```diff
@@ -88,7 +88,13 @@ constructor and destructor.
 
 ## Plugins
 
-TBD
+Plugin lifetime is managed by utilizing piInitializePlugins() and piTearDown().
+GlobalHandler::shutdown() will tear down all registered globals before SYCL RT
+library is unloaded. It will invoke piTearDown() and unload() for each
+plugin. piTearDown() is going to perform any necessary tear-down process at the
+plugin PI level. These two APIs allow on-demand plugin lifetime management. SYCL
+RT can control the beginning and the end of the plugin. 
+![](images/plugin-lifetime.jpg)
```

Comment:
Could you please add arrow with piPluginInit label, which goes from "Plugin Interface" to "Plugin", it should correspond to "bind plugin" arrow?
Also, could you please rename "destory plugin" to piTearDown?